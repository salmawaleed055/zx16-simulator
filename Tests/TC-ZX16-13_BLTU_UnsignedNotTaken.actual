Loaded 12 bytes into memory
0x0000: 0A79    li x1, 5
0x0002: 1471    xori x1, 10
0x0004: 2462    bltu x1, x2, 0x0008
0x0006: 0271    xori x1, 1
0x0008: 0000    add x0, x0
0x000A: FFC7    Unknown opcode

--- Final State ---
x0: 0x0000
x1: 0x000e
x2: 0x0000
x3: 0x0000
x4: 0x0000
x5: 0x0000
a0: 0x0000
a1: 0x0000
PC: 0x000a
---------------------

Simulation finished.
