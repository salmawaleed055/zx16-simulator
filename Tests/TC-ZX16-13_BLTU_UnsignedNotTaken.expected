0x0000: 1471    li x1, 10
0x0002: 0A79    li x2, 5
0x0004: 2462    bltu x1, x2, 0x0008
0x0006: 0271    li x1, 1
0x0008: 0000    add x0, x0
0x000A: FFC7    ecall 0x3FF

--- Final State ---
x0: 0x0000
x1: 0x0001
x2: 0x0005
x3: 0x0000
x4: 0x0000
x5: 0x0000
a0: 0x0000
a1: 0x0000
PC: 0x000C
---------------------
Simulation finished.